test/integration/tests/fip-38-tokemak.ts

```diff
@@ -85,58 +85,4 @@ describe.skip('e2e-fip-38-tokemak', function () {
       ethers.utils.parseEther('10000')
     );
   });
-
-  it('should be able to deposit TOKE, and withdraw', async function () {
```

Comment:
Why remove this test?

Reply:
we don't use a TOKE deposit, the contract has been deployed on mainnet but never used.

the metagovernance PR will use another deposit, so we don't need to keep trace of the existence of this TOKE contract that was never used for anything :p

Reply:
Thanks for explaining